src/PaymentServer/Processors/Stripe.hs

```diff
@@ -58,12 +94,12 @@ getVoucher (MetaData []) = Nothing
 getVoucher (MetaData (("Voucher", value):xs)) = Just value
 getVoucher (MetaData (x:xs)) = getVoucher (MetaData xs)
 
-stripeServer :: VoucherDatabase d => d -> Server StripeAPI
-stripeServer = webhook
+stripeServer :: VoucherDatabase d => d -> ByteString -> Server StripeAPI
```

Comment:
I'd probably lean towards flipping these two args - key, then database.  I'm still trying to get a feel for idiomatic Haskell argument order though.  Key makes sense as an earlier argument, to me, because it is immutable, I guess?  It is also a "simpler" value though I'm not sure I could formally explain "simpler" in this context.  These two observations lead me to think key is more likely to be held constant while the database might vary - so putting key first makes more currying possible.

If you're convinced, flip them.  If not, no worries.

Reply:
Yes, I get what you are saying. Will make that change.

Reply:
Just pushed a commit.